cosmos/bluechip

```diff
@@ -0,0 +1,51 @@
+{
+  "chainId": "bluechip_1",
+  "chainName": "BlueChip",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/bluechip/bluechip.png",
+  "rpc": "http://67.218.8.88:26660/",
+  "rest": "http://67.28.8.88:1318/",
```

Comment:
Could you replace the url scheme to https? It wouldn't work on iOS Keplr Mobile for now.

Reply:
@editaahn I am getting this error "Chain identifier unmatched: (expected: bluechip_1, actual: bluechip)" when trying to run the verification locally. Where is this pulling information from? Our chainId is bluechip_1

Reply:
The chain identifier is `bluechip_1`! Try it for the JSON name and the image directory name.

Reply:
I believe that worked. Although it may be worth changing to bluechip-1 before we add more than our original validator to make things more convenient...

We are just working on getting our restAPIs to sync with the https I will hold off on re submitting until then.